src/ResourceManager/KeyVault/Commands.KeyVault.Test/Scripts/PSHCommon/Common.ps1

```diff
@@ -77,23 +104,21 @@ function Run-Test
     {
          Write-Log "[run-test]: Running test without file comparison"
     }
-        
-    $oldPref = $ErrorActionPreference	 
+
+    $oldPref = $ErrorActionPreference
     $ErrorActionPreference = "SilentlyContinue"
-    #Start-Transcript -Path $transFile	
```

Comment:
Same comment - the ErrorActionPreference setting is not necessary if tthe transcript command is removed.